packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -256,7 +315,18 @@ export const Checkout = ({
           </div>
 
           <div className="subscription-create-footer">
-            {selectedPlan && <TermsAndPrivacy plan={selectedPlan} />}
+            {selectedPlan && (
+              <>
+                <br />
+                <PaymentLegalBlurb provider={undefined} />
```

Comment:
The PayPal privacy policy link doesn't match the link Tony [provided in the ticket](https://mozilla-hub.atlassian.net/browse/FXA-3715?focusedCommentId=441990). It also returns an `Internal Server Error`. Can you update that link?

Not sure how long this has been the case, sadly.